runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -9106,8 +9106,8 @@ done:;
 			goto jni; \
 		case RUN_METHOD_COMPILED: \
 			goto i2j; \
-		DEBUG_ACTIONS \
 		PERFORM_ACTION_VALUE_TYPE_IMSE \
```

Comment:
@gacholio Are you referring to this? or something else?

Reply:
Sorry, missed the fact that you also changed this.